igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitlabCiProperties.java

```diff
@@ -50,9 +50,9 @@ public void setMasters(List<GitlabCiHost> masters) {
     @NotEmpty private String name;
     @NotEmpty private String address;
     private String privateToken;
-    private boolean limitByMembership = false;
-    private boolean limitByOwnership = true;
-    private Integer itemUpperThreshold;
+    private boolean limitByMembership = true;
+    private boolean limitByOwnership = false;
+    private Integer itemUpperThreshold = 1000;
```

Comment:
Adding a default here feels right...and I guess if we're assuming this code didn't work before, then there aren't existing users to worry about.  What happens if there are more than 1000 items though?

Reply:
Same as Jenkins and others (this value gets returned in the overridden method `getPartitionUpperThreshold`).  The new builds gets thrown out and logged.  That lives in the super class method: `CommonPollingMonitor.internalPollSingle()` You can force a fast forward with `curl http://127.0.0.1:8088/admin/pollers/fastforward/gitlabCiBuildMonitor?partition=<name> -X POST`

Reply:
Aaah, ok.  I remember this behavior being a little painful -- at least for docker images...If there are too many new ones, triggering basically stops working until there's manual intervention.  I guess this PR isn't responsible for fixing this...